build/blocks/newsletter-form/src/script.js--output-path.asset.php

```diff
@@ -0,0 +1 @@
+<?php return array('dependencies' => array(), 'version' => 'd8e343f74f68e3e716261c759765b75f');
```

Comment:
Same with this one, seems like it shouldn't have `--output-path.asset.php` at the end of the filename.

Reply:
removed that entire unintended build folder. I had a typo in the package.json; I forgot a space before the `--output-path...`